pkg/strategy/controller.go

```diff
@@ -144,7 +143,6 @@ func normalizeIndexCleaner(spec *v1.JaegerEsIndexCleanerSpec, storage string) {
 		trueVar := true
 		spec.Enabled = &trueVar
 	}
-	spec.Image = util.ImageName(spec.Image, "jaeger-es-index-cleaner-image")
```

Comment:
I am wondering if there should be a null check here.

Reply:
It will never return a null, as if not specified it is initialised from the cli flag, which has a default value.

Reply:
I mean we should have been checking `spec.Image` whether is empty before assigning a value to it.